api/src/main/java/org/opennms/integration/api/v1/config/poller/Parameter.java

```diff
@@ -35,10 +35,12 @@
  * URL to hit is configurable via a parameter. Parameters are specific to the
  * service monitor.
  */
-public interface Parameter{
+public interface Parameter {
 
     String getKey();
 
-    Optional<String> getValue();
+    default Optional<String> getValue() {
+        return Optional.empty();
+    }
```

Comment:
I think this was just optional to support complex values in the core config. As we do not support these, this should be non-optional.